src/test/java/org/openmrs/contrib/qaframework/automation/RegistrationSteps.java

```diff
@@ -13,7 +13,7 @@
 import static org.junit.Assert.assertNotNull;
 
 import org.openmrs.contrib.qaframework.RunTest;
-import org.openmrs.contrib.qaframework.helper.PatientGenerator;
+import org.openmrs.contrib.qaframework.helper.TestData;
 import org.openmrs.contrib.qaframework.helper.TestPatient;
```

Comment:
@Muta-Jonathan  well done, Have you tested whether all the workflows are running perfect locally

Reply:
have tried it but npm not running shows error!

Reply:
@Muta-Jonathan share the logs via pastebin.com

Reply:
@kdaud [here](https://pastebin.com/pKTwdCuU) are the logs

Reply:
this is just about my pc i think